front/app/modules/commercial/content_builder/admin/components/ContentBuilderSettings/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useEditor } from '@craftjs/core';
+import { ROOT_NODE } from '@craftjs/utils';
+import { Box } from '@citizenlab/cl2-component-library';
+import { colors } from 'utils/styleUtils';
+import Button from 'components/UI/Button';
+import messages from '../../messages';
+import { FormattedMessage } from 'utils/cl-intl';
+import { getComponentNameMessage } from '../../utils';
+
+import styled from 'styled-components';
+
+const StyledBox = styled(Box)`
+  box-shadow: -2px 0px 1px 0px rgba(0, 0, 0, 0.06);
+`;
+
+const ContentBuilderSettings = () => {
+  const { actions, selected, isEnabled } = useEditor((state, query) => {
+    const currentNodeId: string = query.getEvent('selected').last();
+    let selected;
+
+    if (currentNodeId) {
+      selected = {
+        id: currentNodeId,
+        name: state.nodes[currentNodeId].data.name,
+        settings:
+          state.nodes[currentNodeId].related &&
+          state.nodes[currentNodeId].related.settings,
+        isDeletable: query.node(currentNodeId).isDeletable(),
+      };
+    }
+
+    return {
+      selected,
+      isEnabled: state.options.enabled,
+    };
+  });
+
+  return selected && isEnabled && selected.id !== ROOT_NODE ? (
+    <StyledBox
+      bgColor={colors.disabledPrimaryButtonBg}
+      px="20px"
+      py="20px"
+      minWidth="400px"
+    >
+      <Box>
+        <Box>
+          <Box pb={'20px'}>
+            <Box>
+              <Box>
+                <h2>
+                  {' '}
+                  <FormattedMessage
+                    {...getComponentNameMessage(selected.name)}
+                  />
+                </h2>
+              </Box>
+            </Box>
+          </Box>
+        </Box>
+        {selected.settings && React.createElement(selected.settings)}
```

Comment:
Normally, we would have a settings component for each craft component where we can customize it (see demo branch). For now, we don't have customization options for the 1 column but I leave it here to be used for the Text component, etc.